plugins/generator-1.16.5/forge-1.16.5/templates/plant.java.ftl

```diff
@@ -454,12 +454,11 @@ import net.minecraft.block.material.Material;
 					}
 				</#if>
 
-				Block block = state.getBlock();
-
+				Block ground = state.getBlock();

```

Comment:
How did this rename fix the bug? :D

Reply:
It doesn't, when I fixed the issue in 1.15 I also changed it in 1.16 but didn't notice there was no issue in that version